php/class-post-type.php

```diff
@@ -115,6 +115,16 @@ public function register() {
 		add_action( 'admin_notices', array( $this, 'show_publish_error_admin_notice' ) );
 		add_action( 'post_submitbox_minor_actions', array( $this, 'hide_disabled_publishing_actions' ) );
 		add_action( 'admin_print_scripts-revision.php', array( $this, 'disable_revision_ui_for_published_posts' ) );
+
+		// Version check for bulk action.
+		if ( version_compare( get_bloginfo( 'version' ), '4.7', '>=' ) ) {
```

Comment:
Nice. Better than using the `$wp_version` global which I usually use. I'll do this in the future.